sim/dex.ts

```diff
@@ -337,7 +337,7 @@ export class ModdedDex {
 	getForme(speciesid: string | Species): string {
 		const id = toID(speciesid || '');
 		const species = this.getSpecies(id);
-		if (species.cosmeticFormes && species.cosmeticFormes.includes(id)) {
+		if (species.cosmeticFormes?.map(toID).includes(id)) {
```

Comment:
```suggestion
		if (species.cosmeticFormes?.includes(species.name)) {
```

Reply:
Wait a minute what is this function even doing? This does exactly the same thing as `getSpecies`, doesn't it?

Reply:
This recommended change makes tests fail.

```diff
1) Dex#getForme
       should handle Flabébé formes:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
+ actual - expected

+ 'Flabébé'
- 'Flabébé-Yellow'
            ^
      + expected - actual

      -Flabébé
      +Flabébé-Yellow

      at Context.<anonymous> (test\sim\dex.js:37:10)
      at processImmediate (internal/timers.js:439:21)
      at process.topLevelDomainCallback (domain.js:126:23)
```

Reply:
Apparently this is a client-only feature

> Dex.getSpecies('flabebeyellow').name
> "Flabébé-Yellow"

The server should probably be updated to match client.

Reply:
Although feel free to leave this alone and do the update in a different commit, I guess.

Reply:
> This recommended change makes tests fail.

Glad I added the test